
package com.postini.pstn.soapapi.v2.automatedbatch;

import javax.xml.ws.WebFault;


/**
 * This class was generated by the JAX-WS RI.
 * JAX-WS RI 2.2.9-b130926.1035
 * Generated source version: 2.2
 * 
 */
@WebFault(name = "UnknownInternalException", targetNamespace = "http://postini.com/PSTN/SOAPAPI/v2/automatedbatch")
public class UnknownInternalException_Exception
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private UnknownInternalException faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public UnknownInternalException_Exception(String message, UnknownInternalException faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param faultInfo
     * @param cause
     * @param message
     */
    public UnknownInternalException_Exception(String message, UnknownInternalException faultInfo, Throwable cause) {
        super(message, cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: com.postini.pstn.soapapi.v2.automatedbatch.UnknownInternalException
     */
    public UnknownInternalException getFaultInfo() {
        return faultInfo;
    }

}
